checkstyleSuppressions.xml

```diff
@@ -21,5 +21,5 @@
     <suppress checks=".*" files=".+[\\/]generated[\\/].+\.java" />
     <suppress checks=".*" files=".+[\\/]generated-sources[\\/].+\.java" />
     <suppress checks=".*" files=".+[\\/]generated-test-sources[\\/].+\.java" />
-
+    <suppress checks="LineLength" files=".+[\\/]jute[\\/].+\.java"/>
```

Comment:
This is added because for code generation we have to write some lines quite long and break them down actually turns into worse readability.